pkg/apis/workload/v1alpha1/types.go

```diff
@@ -112,4 +112,8 @@ const (
 	// AnnotationSkipDefaultObjectCreation is the annotation key for an apiexport or apibinding indicating the other default resources
 	// has been created already. If the created default resource is deleted, it will not be recreated.
 	AnnotationSkipDefaultObjectCreation = "workload.kcp.dev/skip-default-object-creation"
+
+	// InternalSyncTargetPlacementAnnotationKey is a internal annotation key on placement API to mark the synctarget scheduled
+	// from this placement. The value is {location workspace}/{syncTarget name}
+	InternalSyncTargetPlacementAnnotationKey = "internal.workload.kcp.dev/synctarget"
```

Comment:
hmm, I thought this is either a) storing a JSON map with the synctarget per placement, or b) we have multiple annotations, one for each workspace.

Reply:
this annotation is set on placement API. Since each placement only schedule one synctarget (in one selected location), I think we do not need a map. Or if we should set the annotation on the ns, then it should be a JSON map. There is some difference between these two. In the latter approach, if we have two namespaces, they could be scheduled to different synctargetes in a selected location by placement.

Reply:
In the future don't we envision cases when a placement inside a location might result in 2 SyncTargets being scheduled: transparent deployment spreading case for scalability for example ?

Reply:
don't think we will have that beyond temporary during migration. At least I would not expect that being done on purpose by the system.